lib/date_time_parser.ex

```diff
@@ -164,6 +164,20 @@ defmodule DateTimeParser do
     end
   end
 
+  @doc """
+  Parse a `%DateTime{}`, `%NaiveDateTime{}`, `%Date{}`, or `%Time{}` from a string.
+
```

Comment:
Any reason this one has the extra newline and the others don't?

Reply:
no intentional reason. I'll double-check the `mix docs` output and make it consistent either way. Nice catch.

Reply:
Updated to be consistent. I put the sentence about "accepting x options" on a newline for all functions.